app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ActiveRecord::Base
+  has_many :cards, dependent: :destroy
+  
+  validates_presence_of :email, :password
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.